pkg/core/native/native_test/neo_test.go

```diff
@@ -271,6 +271,35 @@ func TestNEO_Vote(t *testing.T) {
 		require.NotEqual(t, candidates[0], pubs[i])
 		require.NotEqual(t, candidates[len(candidates)-1], pubs[i])
 	}
+	//LastGasPerVote should be 0 after unvoting
+	getAccountState := func(t *testing.T, account util.Uint160) (*state.NEOBalance, uint64) {
+		stack, err := neoCommitteeInvoker.TestInvoke(t, "getAccountState", account)
+		require.NoError(t, err)
+		res := stack.Pop().Item()
+		require.Equal(t, 4, len(res.Value().([]stackitem.Item)))
+		as := new(state.NEOBalance)
+		err = as.FromStackItem(res)
+		require.NoError(t, err)
+		return as, res.Value().([]stackitem.Item)[3].Value().(*big.Int).Uint64()
```

Comment:
Second return value is not needed, it's enough that you've checked `require.Equal(t, 4, len(res.Value().([]stackitem.Item)))`.

Reply:
but the `fourth one is zero integer` no need to check it?

Reply:
https://github.com/nspcc-dev/neo-go/blob/9c83beffbb69a8d8a455050cfbba6aeb62d9676b/pkg/core/state/native_state.go#L162-L168